src/Compute/Compute/ChangeLog.md

```diff
@@ -20,6 +20,8 @@
 
 -->
 ## Upcoming Release
+* Added parameter `-EnableHotpatching` to the `Set-AzVMOperatingSystem` cmdlet for Windows machines. 
```

Comment:
I assume there is breaking change in this PR, can you please add a brief description:
"* [Breaking Change] ..."
in the Changelog.md

Reply:
@VeryEarly , the breaking change is the uptake of the new .Net SDK version 44. It changes the response to Get-AzVM for users whitelisted to the VMGuestPAtch API by removing and renaming 4 properties.

Reply:
I'll add that.